apatite/apatite/metrics/gh.py

```diff
@@ -0,0 +1,35 @@
+import glom
+import json
+import os
+import urllib.request
+
+
+DETECT_CMD = 'curl'
+DETECT_ENV = 'GH_API_KEY'
+
+required_cmds = {DETECT_CMD: 'install via your package manager'}
```

Comment:
curl is no longer required, so I'd drop required_cmds or leave it as an empty dict